packages/lib-utils/src/components/label-list/LabelList.tsx

```diff
@@ -29,7 +29,7 @@ export const LabelList: React.FC<LabelListProps> = ({
 }) => {
   const defaultColor = 'purple';
 
-  return _.isEmpty(labels) ? (
+  return _.isEmpty(labels) || !labels ? (
```

Comment:
@kdoberst - curious - what case was `_.isEmpty` missing here?
<img width="220" alt="image" src="https://user-images.githubusercontent.com/21317056/197603945-a7e2b76d-2948-40e7-a7a0-95920c49ea87.png">

Reply:
Down a few lines there is this code:
```
 {Object.entries(labels).map(([key, value]) => (
```

This returns a type error because this line, in theory, would blow up if labels were undefined.